app/views/users/_repos_and_pocs.html.erb

```diff
@@ -0,0 +1,86 @@
+<div class="row repo-show white">
+  <div class="container z-depth-1">
+    <% if @pocs.present? %>
+    <table class="highlight para-text dash-table">
+      <thead>
+        <tr>
+          <th>No.</th>
+          <th data-field="id">Name</th>
+          <th data-field="price">Downloads</th>
+          <th data-field="price">Views</th>
+          <th data-field="price">Favourites</th>
+          <th data-field="price">Show/Hide</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @pocs.each.with_index do |poc, index| %>
+        <tr>
+          <td><%= index + 1 %></td>
+          <td><%= poc.name %></td>
+          <td><%= poc.no_of_downloads.to_i %></td>
+          <td><%= poc.impressions.count.to_i %></td>
+          <td><%= poc.favourites.count.to_i %></td>
+          <td>
+            <div class="switch">
+              <label>
+                <input type="checkbox" onchange="change_hide('<%= hide_repository_path(poc.id) %>')" <%= "checked" if poc.hide %>>
+                <span class="lever"></span>
+              </label>
+            </div>
+          </td>
+          <td data-field="price">
+            <%= link_to edit_repository_path(poc.id) do %>
+            <i class="material-icons">mode_edit</i>
+            <% end %>
+            <%= link_to repository_path(poc.id), method: :delete, remote: true, data: { confirm: 'Are you sure?' } do %>
+            <i class="material-icons">delete</i>
+            <% end %>
+          </td>
+        </tr>
+        <% end %>
+
+      </tbody>
+    </table>
+  <% else %>
+    <b>No POCs to show.</b>
+    <% end %>
+  </div>
+
+  <br>
+
+  <div class="container z-depth-1">
+    <% if @repositories.present? %>
+    <table class="highlight para-text dash-table">
+      <thead>
+        <tr>
+          <th>No.</th>
+          <th data-field="id">Name</th>
+          <th data-field="price">Add Repo</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% index = 1 %>
+        <% @repositories.each do |repo| %>
```

Comment:
@AmitPatel-BoTreeConsulting inside this loop, there is one more condition which affect index, so it is not possible to take each_with_index